src/templates/static/metatrader/download.jsx

```diff
@@ -1,64 +1,111 @@
 import React from 'react';
 import SeparatorLine from '../../_common/components/separator_line.jsx';
 
-const DownloadBlock = ({
-    badge,
-    desc,
-    download,
-    href,
-    image,
-    os,
-    target,
-    title,
-}) => (
+const DownloadBlock = ({ badge, desc, download, href, image, os, target, title }) => (
```

Comment:
if there are four or more arguments we break it into multiline, you can find it here https://github.com/binary-com/binary-static/blob/master/src/templates/README.md#jsx-rules